api/http/handler/users/user_helm_repos.go

```diff
@@ -36,13 +36,13 @@ func (p *addHelmRepoUrlPayload) Validate(_ *http.Request) error {
 // @security jwt
 // @accept json
 // @produce json
-// @param id path int true "Environment(Endpoint) identifier"
+// @param id path int true "User identifier"
 // @param payload body addHelmRepoUrlPayload true "Helm Repository"
-// @success 200 {object} portainer.HelmUserRepository "Success"
+// @success 200 {object} portaineree.HelmUserRepository "Success"
```

Comment:
This is wrong.  It should be `portainer.HelmUserRepository` in the CE codebase, otherwise make docs will fail